src/man/sssd.conf.5.xml

```diff
@@ -988,6 +988,24 @@ fallback_homedir = /home/%u
                     </listitem>
                 </varlistentry>
                 <varlistentry>
+                    <term>use_memcache (bool)</term>
+                    <listitem>
+                        <para>
+                            Whether to use in-memory cache to improve
+                            performance.
+                        </para>
+                        <para>
+                            Default: True
+                        </para>
+                        <para>
+                            NOTE: If the environment variable
+                            SSS_NSS_USE_MEMCACHE is set to "NO", client
+                            applications will not use the fast in-memory
+                            cache.
```

Comment:
Should the environment variable override this option if set or vice versa? Either way, one of these if set should override the other and it should be documented.

Reply:
For example, if `SSS_NSS_USE_MEMCACHE` is set to "YES"/"NO" it will be always enabled/disabled regardless of `use_memcache` option (or the other way around).